openapi-cli/src/main/java/io/ballerina/openapi/generators/client/BallerinaUtilGenerator.java

```diff
@@ -170,6 +180,7 @@ public SyntaxTree generateUtilSyntaxTree() throws IOException {
         }
         if (queryParamsFound) {
             functionNameList.addAll(Arrays.asList(
+                    CREATE_FORM_URLENCODED_REQUEST_BODY, GET_FORM_STYLE_REQUEST,
```

Comment:
Not getting the reason to generate `CREATE_FORM_URLENCODED_REQUEST_BODY`. `GET_FORM_STYLE_REQUEST` is already in the list

Reply:
Fixed